DataFormats/EgammaCandidates/interface/AODHIPhoton.h

```diff
@@ -0,0 +1,72 @@
+#ifndef AODHIPhoton_h
+#define AODHIPhoton_h
+
+#include "DataFormats/EgammaCandidates/interface/Photon.h"
+
+namespace aod{
+
+  class AODHIPhoton : public reco::Photon {
+
+  public:
+
+  AODHIPhoton() : reco::Photon() {}
+    virtual ~AODHIPhoton() {}
+
+  AODHIPhoton(const reco::Photon& p) : reco::Photon(p) {}
+
+    float cc1() {return cc1_;}
```

Comment:
looking at implementation, to correct myself, these are not cones, these are eta slices